handbook/growth.md

```diff
@@ -0,0 +1,86 @@
+
+### Press release boilerplate
+
+Fleet gives teams fast, reliable access to data about the production servers, employee laptops, and other devices they manage - no matter the operating system. Users can search for any device data using SQL queries, making it faster to respond to incidents and automate IT. Fleet can also be used to monitor vulnerabilities, battery health, software, and even EDR and MDM tools like Crowdstrike, Munki, Jamf, and Carbon Black, to help confirm that those platforms are working how administrators think they are. Fleet is open-source software. It's easy to get started quickly, easy to deploy, and it even comes with an enterprise-friendly free tier available under the MIT license.
+
+IT and security teams love Fleet because of its flexibility and conventions. Instead of secretly collecting as much data as possible, Fleet defaults to privacy and transparency, capturing only the data your organization needs to meet its compliance, security, and management goals, with clearly-defined, flexible limits.   
+
+That means better privacy. Better device performance. And better data, with less noise.
+
+## Sponsoring events
+
+When reaching out for sponsorships, Fleet's goal is to expose potential hires, contributors, and users to Fleet and osquery.
+Track prospective sponsorships in our [partnerships and outreach Google Sheet:](https://docs.google.com/spreadsheets/d/107AwHKqFjt7TWItnf8pFknSwwxb_gsp6awB66t7YE_w/edit#gid=2108184225)
+
+Once a relevant sponsorship opportunity and its prospectus are reviewed:
+1. Create a new [GitHub issue](https://github.com/fleetdm/fleet/issues/new).
+ 
+2. Detail the important information of the event, such as date, name of the event, location, and page links to the relevant prospectus. 
+ 
+3. Add the issue to the “Conferences/speaking” column of the [Growth plan project](https://github.com/orgs/fleetdm/projects/21).
+ 
+4. Schedule a meeting with the representatives at the event to discuss pricing and sponsorship tiers.
+ 
+5. Invoices should be received at billing@fleetdm.com and sent to Eric Shaw for approval.
+ 
+6. Eric Shaw (Business Operations) will route signatures required over to Mike McNeil (CEO) with Docusign.
+ 
+7. Once the above steps are completed, use the [Speaking events issue template](https://github.com/fleetdm/confidential/issues/new?assignees=mike-j-thomas&labels=&template=6-speaking-event.md&title=Speaking+event) to prepare speakers and participants for the event.
+
+## Slack channels
+
+These are the Slack channels the growth team maintains. If the channel has a [directly responsible individual](./people.md#directly-resonsible-individuals) (**DRI**), they will be specified. These people are responsible for keeping up with all new messages, even if they aren't mentioned. 
+
+- **#g-growth** **DRI**: Tim Kern
+- **#help-swag** **DRI**: Drew Baker
+- **#help-promote**
```

Comment:
Should we include this without the responsible party?

Reply:
@Desmi-Dizney as I understand it every channel doesn't have a DRI. I'm trying to figure out if it does have a DRI but should we assume it's no one for the time being and then correct that later if it's wrong?

Reply:
Was able to figure out that I'd be a good DRI for this, so I'll adjust that